python_logging_rabbitmq/formatters.py

```diff
@@ -27,4 +34,10 @@ def format(self, record):
         if 'exc_info' in data and data['exc_info']:
             data['exc_info'] = self.formatException(data['exc_info'])
 
+        if self.include:
+            data = {f: data[f] for f in self.include}
```

Comment:
@IwoHerka are you assuming that `KeyError` is reasonable when someone will provide invalid include dict?

Reply:
Yes, it's a matter of configuration. Configuration is not validated anywhere, I see no reason to validate it here.

Reply:
Sound. Approved.